zndraw/static/pycom/Cache.js

```diff
@@ -128,6 +129,12 @@ class Cache {
         this.world.setStep(this.world.getStep());
       }
     }
+    indicesToDelete.sort((a, b) => b - a);
+    for (const index of indicesToDelete) {
+      console.log("deleting", index);
+      delete this._cache[index];
+    }
```

Comment:
Is this working correctly? Isn't `this._cache` a dict
https://github.com/zincware/ZnDraw/blob/d395afebcc89b5a4249c4ddbdb98433345679d97/zndraw/static/pycom/Cache.js#L89
and therefore we should run a final script "closing" the gaps?